templates/field/field--node--oe-organisation-staff-link--oe-organisation--full.html.twig

```diff
@@ -0,0 +1,21 @@
+{#
+/**
+ * @file
+ * Theme override for Organisation staff search link field template in the Organisation CT full view mode.
+ *
+ * @see ./core/themes/stable/templates/field/field.html.twig
+ */
+#}
+{% for item in items %}
+  <div class="ecl-u-mt-s">
+    {% include '@ecl-twig/link' with {
+      link: {
+        type: 'standalone',
+        label: item.content['#title'],
+        path: item.content['#url'],
```

Comment:
By accessing link render array portions we don't allow cache to bubble up. The easiest way to fix it is to render the link and assign the output to a catch all variable, that will never be displayed, like that:

```twig
{% set bubble_cache = item|render %}
```